helm/app-operator/values.yaml

```diff
@@ -0,0 +1,34 @@
+# Default values for app-operator-chart.
```

Comment:
```suggestion
# Default values for app-operator chart.
```

Reply:
This needs updating because the chart was renamed.

Reply:
https://github.com/giantswarm/app-operator/pull/240/files#diff-540ebd3b9c05ac90abbd6b0cb458468eR1 This is there already, I don't know why github didn't show it as outdated :(.

Reply:
Strange. Thanks for confirming. 👍